src/test/java/org/opentripplanner/openstreetmap/wayproperty/specifier/BestMatchSpecifierTest.java

```diff
@@ -21,13 +19,6 @@ class BestMatchSpecifierTest extends SpecifierTest {
   );
 
   static OsmSpecifier bikeLane = new BestMatchSpecifier("highway=residential;cycleway=lane");
-  static OsmSpecifier regularCobblestones = new BestMatchSpecifier(
-    "highway=residential;surface=cobblestones"
-  );
-  static OsmSpecifier cobblestonesFlattened = new BestMatchSpecifier(
-    "highway=residential;surface=cobblestones:flattened"
```

Comment:
I'm happy this goes away. I kept it for backwards-compatibility but always had the feeling that it's a strange feature.